src/components/ErrorPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Container } from 'reactstrap';
+import { FormattedMessage } from 'react-intl';
+import { Link } from 'react-router-dom';
+
+import apiClient from '../config/apiClient';
+
+export default class ErrorPage extends Component {
+  componentDidMount() {}
+
+  renderLink() {
+    const { username } = apiClient.getAuthenticationState().authentication;
+    return <Link to={`/u/${username}`}>here</Link>;
+  }
+
+  render() {
+    return (
+      <Container
+        fluid
+        className="d-flex py-5 justify-content-center align-items-start text-center"
+      >
+        <p
+          className="my-0 py-5 text-muted"
+          style={{ maxWidth: '32em' }}
+        >
+          <FormattedMessage
+            id="profile.error.message"
+            defaultMessage="An unexpected error occurred. Please click {here} to try again."
```

Comment:
Nit: I wonder if `{profileLink}` might be more clear, and might affect translation of the rest of the sentence(?) in some languages.